README.md

```diff
@@ -18,6 +18,15 @@ have feedback you'd like to share with us.
 If you'd like to follow along, [CHANGELOG.md](CHANGELOG.md) is kept up to date
 with each commit.
 
+## Getting Started
+This readme assumes you have an environment configured with AWS credentials and IAM users/roles with appropriate access to QLDB. This readme focuses on shell configuration and commands. For a guide about prerequisites and general use of the shell, please see the developer guide: https://docs.aws.amazon.com/qldb/latest/developerguide/data-shell.html
+
+## QLDB Core Concepts
+QLDB provides guarantees about consistency during transactions, and ensures that transactions are consistent upon commit via Optimistic Currency Control. This applies to all queries, so when using the shell, your statements must be run as transactions. By default, the shell enables auto-commit mode, which will interpret queries by default as transactions, meaning you do not have to `start transaction` and `commit` manually each time. This is configurable (see below).
+
+Additionally, currently, transactions have a 30-second window, so when using the shell, you'll want to submit queries within that window. Otherwise, you'll get an error and need to retry.
```

Comment:
There's probably a better way to say this. I wanted to call it out clearly because there were a few times I was getting errors for transactions expiring as I was alt-tabbing or constructing my query. It's a little tough to find in the dev guide, so I figured we may want to call it out ahead of time. This might also be a place to reinforce *why* that limit exists.

Alternatively instead of raising this specific issue, we could reference the [Common Errors](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-errors.html) which is currently nested under the "Drivers" section.

Reply:
Definitely helpful to call out the transaction timeout limit. How about this verbiage directly from the *Optimizing query performance* topic in the dev guide? Feel free to tweak this as you see fit for the shell:

>Transactions are subject to a [transaction timeout limit](https://docs.aws.amazon.com/qldb/latest/developerguide/limits.html#limits.fixed). A transaction can run for up to **30 seconds** before being committed. After this limit, QLDB rejects any work done on the transaction and discards the [session](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-session-management.html) that ran the transaction. This limit protects the service's client from leaking sessions by starting transactions and not committing or canceling them.

Reply:
For something more specific to the shell, there's also this note from the [shell dev guide topic](https://docs.aws.amazon.com/qldb/latest/developerguide/data-shell.html#data-shell-transactions) which I shortened a bit here:

>An interactive transaction adheres to QLDB's [transaction timeout limit](https://docs.aws.amazon.com/qldb/latest/developerguide/limits.html#limits.fixed). If you don't commit a transaction within **30 seconds** of starting it, QLDB automatically expires the transaction and rejects any changes made during the transaction.
>
>Then, the shell displays an error message and returns to the normal command prompt. To retry, you must enter the `begin` or `start transaction` command again to begin a new transaction.